jobs/mysql/spec

```diff
@@ -247,3 +247,12 @@ properties:
   cf_mysql.mysql.enable_galera:
     default: true
     description: 'Advanced feature. Only use this when deploying a single MySQL node with no intention to run any other components of cf-mysql-release.'
+
+  cf_mysql.mysql.long_query_time:
+    default: 30
+    description: 'If a query takes longer than this many seconds, the server increments the Slow_queries status variable. If the slow query log is enabled, the query is logged to the slow query log file'
+
```

Comment:
suggesting a user-facing wording such as 

"Threshold in seconds above which SQL queries get logged in the slow query log file"

Reply:
If you want , the description I have written is the official one.